src/bridge.cpp

```diff
@@ -12,6 +12,7 @@ namespace cura
 
 int bridgeAngle(const Settings& settings, const Polygons& skin_outline, const SliceDataStorage& storage, const unsigned layer_nr, const unsigned bridge_layer, const SupportLayer* support_layer, Polygons& supported_regions)
 {
+    assert(skin_outline.size() > 0);
```

Comment:
Why not use '! skin_outline.empty()'  it is not guaranteed that size() is a constant-time function, and it is generally more efficient and also shows clearer intent to use empty().